bucket/winpython.json

```diff
@@ -1,20 +1,27 @@
 {
     "description": "Free, open-source and portable Python distribution for Windows",
-    "version": "3.6.7.0",
+    "version": "3.7.1.0",
     "license": "MIT",
+    "homepage": "https://winpython.github.io/",
     "architecture": {
         "64bit": {
-            "url": "https://downloads.sourceforge.net/project/winpython/WinPython_3.6/3.6.7.0/WinPython64-3.6.7.0Zero.exe#/dl.7z",
-            "hash": "e52e2606f3a2f7d79fd9eac0701b144efe0832ecdcf7c5a2f3bf3aebba4e70c6",
-            "extract_dir": "python-3.6.7.amd64"
+            "url": "https://downloads.sourceforge.net/project/winpython/WinPython_3.7/3.7.1.0/WinPython64-3.7.1.0Zero.exe",
+            "hash": "4e81abd373c089cd9b6687393bb3d179e6792a97d8f088f77d35b9067f118c0f"
         },
         "32bit": {
-            "url": "https://downloads.sourceforge.net/project/winpython/WinPython_3.6/3.6.7.0/WinPython32-3.6.7.0Zero.exe#/dl.7z",
-            "hash": "8b6b22f36aecb80ce8b13212deb4bbba0e9a902ddd081d7c6288cf0f44e419be",
-            "extract_dir": "python-3.6.7"
+            "url": "https://downloads.sourceforge.net/project/winpython/WinPython_3.7/3.7.1.0/WinPython32-3.7.1.0Zero.exe",
+            "hash": "a09c79c7630278f831d257c03bab99f76d3a0e23f8b4362a3e1e2495a3aa088f"
         }
     },
-    "homepage": "https://winpython.github.io/",
+    "innosetup": true,
+    "installer": {
+        "script": [
+            "Get-ChildItem \"$dir\" -Exclude 'python-*' | Remove-Item -Force -Recurse",
+            "$fold = \"$dir\\\" + (Get-ChildItem \"$dir\")[0]",
```

Comment:
This would be evaluated as `someDir" + (Get-ChildItem "$dir")[0])`

Reply:
Just changed it to: `$fold = \"$dir\\\" + ((Get-ChildItem \"$dir\")[0])`

Reply:
Sorry. Misslooked one quotation marks.

Reply:
Great, thanks for the advice!

Reply:
Ok, it should be fine now

Reply:
Still seeing url instead of github

`"github": "https://github.com/winpython/winpython/"`

Reply:
As you said, if I use github it fails in getting the latest version, since their tags do not match the software's version.
i.e. it matches 3.6.7.0, while the latest version is 3.7.1.0.
They're both maintained, but I guess we should stick to the latest stable version, right?

Reply:
Oh, I see, the order of the assets is lexicographic in the tag's url (https://github.com/winpython/winpython/releases/tag/1.11.20181031), while it's not in the release url (https://github.com/winpython/winpython/releases) so the first occurrence is different.

Is there a way to match the occurrence with the greater number (numerical sorted)?

Reply:
What?? No.

Just change `"url": "github.com/.../releases/latest/"` to `"github": "https://github.com/winpython/winpython/"` It's same, but if github change their URLs it will be easier to change one file, instead of bunch of manifests.

Reply:
I see that would be the best practice, so we stick to the previous version (3.6.7.0)?
![cattura](https://user-images.githubusercontent.com/34317613/49697719-197eda00-fbbb-11e8-9601-13a319c0dcd6.PNG)

Reply:
Weird. But easily fixable.

Add reverse: true

![proof](https://i.imgur.com/zCOKGkb.png)

Reply:
Cool! Let's hope it'll remain consinstent with the future releases 😆

Reply:
Can be condensed to:

```json
    "installer": {
        "script": [
            "Get-ChildItem \"$dir\" -Exclude 'python-*' | Remove-Item -Force -Recurse",
            "Move-Item \"$dir\\python-*\\*\" \"$dir\"",
            "Remove-Item 'python-*'"
        ]
    },
```

Reply:
Good, I changed it accordingly